docs/charts/profile_columns.ipynb

```diff
@@ -0,0 +1,551 @@
+       "</script>"
+      ],
+      "text/plain": [
+       "alt.VConcatChart(...)"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import altair as alt\n",
+    "from splink.datasets import splink_datasets\n",
+    "from splink.duckdb.linker import DuckDBLinker\n",
+    "\n",
+    "df = splink_datasets.historical_50k\n",
+    "linker = DuckDBLinker(df)\n",
+    "\n",
+    "linker.profile_columns(\n",
+    "    [\"first_name\", \"gender\", \"birth_place\", \"dob\"]\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### What the chart shows\n",
+    "\n",
+    "The `profile_columns` chart shows 3 charts for each selected column:\n",
+    "\n",
+    "- The left chart shows the distribution of all values in the column.\n",
+    "- The middle chart shows the counts of the ten most common values in the column.\n",
+    "- The right chart shows the counts of the ten least common values in the column."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### How to interpret the chart\n",
+    "\n",
+    "The distribution of values in your data is important for two main reasons:\n",
+    "\n",
+    "1. Columns with higher cardinality (number of distinct values) are usually more useful for data linking. For instance, date of birth is a much stronger linkage variable than gender.\n",
+    "\n",
+    "2. The skew of values is important. If you have a birth_place column that has 1,000 distinct values, but 75% of them are London, this is much less useful for linkage than if the 1,000 values were equally distributed"
```

Comment:
I'd be tempted to bold `birth_place` here

Reply:
done and pushed